soc/arm/atmel_sam/sam4e/Kconfig.series

```diff
@@ -2,14 +2,18 @@
 
 # Copyright (c) 2017 Justin Watson
 # Copyright (c) 2018 Vincent van der Locht
-# Copyright (c) 2019 Gerson Fernando Budke
+# Copyright (c) 2019-2020 Gerson Fernando Budke <nandojve@gmail.com>
 # SPDX-License-Identifier: Apache-2.0
 
 config SOC_SERIES_SAM4E
 	bool "Atmel SAM4E MCU"
 	select ARM
+	select ARM_MPU
 	select CPU_CORTEX_M4
+	# select CPU_CORTEX_M_HAS_DSP
```

Comment:
Until the ARM FP symbols are refactored, for now, this can be thought of as implied when `CPU_HAS_FPU` is selected for the Cortex-M4 and Cortex-M7 (ARMv7E-M).

Reply:
Done!